### PR TITLE
Spreadsheet Settings

### PR DESCRIPTION
**Pull request by joshholat**
Screenshot 2019-10-16 13 19 12

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -1075,3 +1075,60 @@ See [Create Report Template](#create-report-template) for details on format and
...
+                "top_level_id": 18009,
+                "selected_id": 18027
+            }
+        ]
+    }
+}
+```
+
+This endpoint fetches default values to use in various cases for the user in the context
+of a spreadsheet (i.e. inside of Cube add-ons).
```
Comment (DNU-bennett) [inline source/index.html.md]: @joshholat If I'm an outsider or new to the code base and reading this, I wouldn't know what this means or why I'd use it. 

Maybe say something about how this endpoint returns reasonable defaults when the user hasn't specified dimensions to use for rows, columns, filters

**Review by DNU-bennett — CHANGES_REQUESTED** [review]
Docs could address __why__ this endpoint exists in the first place. See my comment

File: source/index.html.md (hunk 1 above)
Comment (joshholat) [reply source/index.html.md]: Updated!

**Review by DNU-bennett — APPROVED** [review]
:mahjong:

**Pull request merged**